packages/graphql/fragments/postFieldsFragment.js

```diff
@@ -134,12 +110,6 @@ export default gql`
       position
       id
     }
-    postMemberships {
```

Comment:
I would keep this for now as this fragment is used throughout, and it has caching benefits to query a post using the same fragment as much as possible... If it is making a non-trivial difference in the query time to remove this postMemberships id / group.id part then it is worth looking at more carefully, but my suspicion that this particular part isn't changing these substantively. You can try URQL dev tools to get a read on the actual query times in dev to compare.

Reply:
but postMemberships are not used anywhere in the code base

Reply:
Loren, does this get used by urql to key a relationship between posts and memberships that we want to track? Doing a quick scan of the monorepo, it doesn't seem to be used explicitly

Reply:
I guess not, totally good removing them then.